t/lib/MemcachedTest.pm

```diff
@@ -255,8 +262,9 @@ sub is_running {
 }
 
 sub new_memcached {
-    my ($args, $passed_port) = @_;
+    my ($args, $passed_port, $passed_non_debug_build_flag) = @_;
     my $port = $passed_port;
+    my $non_debug_build_flag = $passed_non_debug_build_flag;
```

Comment:
don't need this line; just call the above variable non_debug_build_flag. I think $port is being re-assigned because of some old code (it gets re-written sometimes)

Reply:
fixed it.